programmers/다트.js

```diff
@@ -0,0 +1,40 @@
+function solution(dart) {
+    
+    let num = 0;
+    let answer = [];
+    let temp = 0;
```

Comment:
```suggestion
    const answer = [];
    let num = 0;
    let temp = 0;
```
answer 자체를 바꿀 생각이 아니라면 const로 배열 선언 해주세요!
그리고 순서는 보기 편하게 const  let 순으로 넘어갑니다